integration/simulation/network/inmemory.go

```diff
@@ -33,6 +36,12 @@ func (n *basicNetworkOfInMemoryNodes) Create(params *params.SimParams, stats *st
 	obscuroNodes := make([]*host.Node, params.NumberOfNodes)
 	n.obscuroClients = make([]obscuroclient.Client, params.NumberOfNodes)
 
+	// Invent some addresses to assign as the L1 erc20 contracts
+	a := common.HexToAddress("AA")
```

Comment:
As above - if these are dummy addresses, can we rename them to make that clearer? e.g. `dummyBTCAddress` and `dummyObsAddress`.